pages/lib/refinery/pages/engine.rb

```diff
@@ -49,14 +52,13 @@ def append_marketable_routes
         Refinery::Core::Engine.routes.append do
           get '*path', :to => 'pages#show', :as => :marketable_page
         end
-        Rails.application.routes_reloader.reload!
       end
 
       # Add any parts of routes as reserved words.
       def add_route_parts_as_reserved_words
         ActiveSupport.on_load(:active_record) do
           # do not add routes with :allow_slug => true
-          included_routes = Rails.application.routes.named_routes.routes.reject{ |name, route| route.defaults[:allow_slug] }
+          included_routes = Rails.application.routes.named_routes.to_a.reject{ |name, route| route.defaults[:allow_slug] }
```

Comment:
I understand why the `.to_a` was added, but how come the `.routes` part was dropped?